moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -338,6 +342,21 @@ def planning_scene_monitor(
         }
         return self
 
+    def sensors_3d(self, file_path: Optional[str] = None):
+        """Load sensors_3d paramerss.
+
+        :param file_path: Absolute or relative path to the sensors_3d yaml file (w.r.t. robot_name_moveit_config).
+        :return: Instance of MoveItConfigsBuilder with robot_description_planning loaded.
+        """
+        sensors_path = self._package_path / (
+            file_path or self.__config_dir_path / "sensors_3d.yaml"
+        )
+        if sensors_path.exists():
+            sensors_data = load_yaml(sensors_path)
+            if len(sensors_data["sensors"]) > 0 and sensors_data["sensors"][0]:
```

Comment:
the second part of this (that the first sensor is non-empty) can be removed once https://github.com/ros-planning/moveit_resources/pull/141 is released